pages/mentee/_slug.vue

```diff
@@ -29,6 +18,7 @@
 export default {
   async asyncData ({ $content, params, error }) {
     const [doc] = await $content('mentees').where({ slug: { $eq: params.slug } }).fetch()
+    doc.picture = await fetch(`https://api.github.com/users/${doc.github.substring(19)}`).then(res => res.json()).then(data => data.avatar_url)
```

Comment:
GitHub API'a gidip avatar_url'i almak yerine direkt kullanicinin github handle'indan url'i uretemez miyiz?

`https://github.com/${doc.github.split('/').pop()}.png?size=200`

Reply:
Cok daha kolay ve güzel olur 😄 Boyle alabilcegimizi bilmiyordum

Reply:
Tabii ki github url'i varsa bunu yapmaliyiz.
Mumkunse img'in basilacagi yerde github url'i varsa avatar yoksa thumbnail olmali

Reply:
`    doc.picture = doc.github ? `https://github.com/${doc.github.split('/').pop()}.png?size=200` : 'https://fishgate.co.za/wp-content/uploads/2019/08/person-placeholder.jpg'
` 
bu sekilde eger github linki varsa avatarı yoksa onun yerine bi placeholder urki atadım. Bu sekilde sorun cözülüyor ne dusunuyosunuz?

Reply:
Eger kisinin github'i yoksa isminin ve soyisminin bas harflerinden avatar ureten bir site var oraya fallback yapalim mi:

`url = 'https://ui-avatars.com/api/?name=' + name`

Reply:
Harika fikir, cok daha güzel olur. 
` doc.picture = doc.github ? https://github.com/${doc.github.split('/').pop()}.png?size=200 : https://ui-avatars.com/api/?name=${doc.name}
`
Son hali bu sekilde.

Reply:
Ben datayi uretirken direkt saglikli uretecegim avatar diye bir degisken gelecek sen bu sekilde iletirsen harika olur

Reply:
yani picture yerine avatar'ı kullanıyoruz di mi?

Reply:
evet doc.avatar geldigini dusun, birazdan master'da olmus olacak